coins.json

```diff
@@ -648,6 +648,6 @@
         "base58Hasher": "sha256d",
         "xpub": "xpub",
         "xprv": "xprv",
-        "explorer": "https://ravencoin.network/tx/"
+        "explorer": "https://blockbook.ravencoin.org/tx/"
```

Comment:
which explorer one is more user friendly?

Reply:
`https://blockbook.ravencoin.org/tx/` is better, as it displays all the details on one page, also displays tx raw hex, whereas for `https://ravencoin.network/tx/`, we need to use `https://ravencoin.network/api/tx/` to view the details, and it doesn't display tx raw hex.

e.g.
https://ravencoin.network/api/tx/3717b528eb4925461d9de5a596d2eefe175985740b4fda153255e10135f236a6

Reply:
Yes, but for end user, https://ravencoin.network/tx/ might be better, user don't have to know rawtx

Reply:
right, actually it was updated as per @Scotty0448's comment
https://github.com/TrustWallet/wallet-core/pull/484#issuecomment-498040621

Reply:
Let’s keep the initial one

Reply:
Done in 000314fbc4cc9a3e2cc79d5b8fe53f1d2329dbc1